github-actions/runtime-variables/RuntimeVariables.MD

```diff
@@ -0,0 +1,234 @@
+# Runtime Variables in Github Actions
+
+## Objective
+While Github Actions is a popular choice for writing and running CI/CD pipelines, especially for open source projects hosted on Github, it lacks specific quality of life features found in other CI/CD environments. One key feature that Github Actions has not yet implemented is the ability to **mock and inject runtime variables** into a workflow, in order to test the pipeline itself.
```

Comment:
I would make the comparisson here that we are attempting to bridge a feature that exists in DevOps to a feature that doesn't exist in GitHub in a conceptually similar way.

Reply:
Done!